homeassistant/components/azure_data_explorer/client.py

```diff
@@ -0,0 +1,79 @@
+"""Setting up the ingest client."""
+from __future__ import annotations
+
+import io
+import logging
+
+from azure.kusto.data import KustoClient, KustoConnectionStringBuilder
+from azure.kusto.data.data_format import DataFormat
+from azure.kusto.ingest import (
+    IngestionProperties,
+    ManagedStreamingIngestClient,
+    QueuedIngestClient,
+    StreamDescriptor,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+# Suppress very verbose logging from client
+logger = logging.getLogger("azure")
+logger.setLevel(logging.WARNING)
+
+
+class AzureDataExplorerClient:
+    """Class for Azure Data Explorer Client."""
+
+    def __init__(self, **data) -> None:
+        """Create the right class."""
+
+        self.cluster_ingest_uri = data["cluster_ingest_uri"]
```

Comment:
Please use the constants like `CONF_ADX_CLUSTER_INGEST_URI`

Reply:
Where ? 
do you mean:
self._CONF_ADX_CLUSTER_INGEST_URI = data["cluster_ingest_uri"]

Reply:
i fixed it